pop.js

```diff
@@ -0,0 +1,349 @@
+const workSection = document.getElementById('work-section');
+const modal = document.getElementById('modals-container');
+
+const cardInfos = {
+  title: [
+    'Multi-Post Stories',
+    'Data Dashboard Healthcare',
+    'Website Portfolio',
+    'Profesional Art Printing Data More',
+    'My Recent Works',
+  ],
+  image: [
+    'images/ImgPlaceholder0.svg',
+    'images/ImgPlaceholder1.svg',
+    'images/ImgPlaceholder2.svg',
+    'images/ImgPlaceholder3.svg',
+    'images/ImgPlaceholder4.svg',
+    'images/Cancel.svg',
+    'images/liveIcon.svg',
+    'images/githubIcon.svg',
+  ],
+  description: [
+    "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
```

Comment:
Good job working on the modal for your portfolio, however, taking a critical look at your implementation, it is not the best since it is repetitive and as a developer, reducing the amount of code to improve readability and reduce repetition is paramount. If you had created an array of objects as required in the project [description](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20need%20to%20use%20a%20JavaScript%20array%20to%20store%20all%20of%20the%20information%20for%20all%20projects.), if it was approached that way, a single loop through that array would create the cards and the modal and it will eliminate the scenario of repeating yourself. So, I recommend updating this to an array of objects.

Reply:
Could you please tell me about solving this issue with some more details because I din't get your point.